recipes/r-abcrf/meta.yaml

```
@@ -0,0 +1,115 @@
+{% set version = '1.9' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-abcrf
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/abcrf_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/abcrf/abcrf_{{ version }}.tar.gz
+  sha256: b1eb36bd10dd71ede3274dcd243131b79fee32fb385c8b4d7715dcfe8c0ab022
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-mass
+    - r-rcpp >=0.11.2
+    - r-rcpparmadillo
+    - r-doparallel
+    - r-foreach
+    - r-matrixstats
+    - r-ranger
+    - r-readr
+    - r-stringr
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-mass
+    - r-rcpp >=0.11.2
+    - r-rcpparmadillo
+    - r-doparallel
+    - r-foreach
+    - r-matrixstats
+    - r-ranger
+    - r-readr
+    - r-stringr
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('abcrf')"           # [not win]
+    - "\"%R%\" -e \"library('abcrf')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://CRAN.R-project.org/package=abcrf
+  license: GPL-2.0-or-later
+  summary: Performs Approximate Bayesian Computation (ABC) model choice and parameter inference
+    via random forests. Pudlo P., Marin J.-M., Estoup A., Cornuet J.-M., Gautier M.
+    and Robert C. P. (2016) <doi:10.1093/bioinformatics/btv684>. Estoup A., Raynal L.,
+    Verdu P. and Marin J.-M. <http://journal-sfds.fr/article/view/709>. Raynal L., Marin
+    J.-M., Pudlo P., Ribatet M., Robert C. P. and Estoup A. (2019) <doi:10.1093/bioinformatics/bty867>.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

Include alternative license:

```suggestion
     - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
     - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```